docs/guide/acquiring_data/helioviewer.rst

```diff
@@ -160,7 +160,7 @@ In this example we will query Helioviewer for the relevant JPEG 2000 file closes
    >>> data_sources = hv.get_data_sources()  # doctest: +REMOTE_DATA
    >>> filepath = hv.download_jp2('2012/07/05 00:30:00', observatory='SDO',
    ...                            instrument='HMI', measurement='continuum')  # doctest: +REMOTE_DATA
-   >>> hmi = Map(filepath)  # doctest: +REMOTE_DATA
+   >>> hmi = Map(filepath)  # doctest: +SKIP
```

Comment:
Due to moving to tox via pip, we can not expect openjpeg to be installed.

Reply:
:(

Reply:
I did add it them to macos and linux CI, but not been able to find a way to do it for windows.